inject-dist/observers.js

```diff
@@ -105,12 +119,21 @@ function initializeObserver(pathname) {
           }
           stackCounter[stack]++;
         }
-
+        if (pathname in FUNCTION_CALL_ARGUMENTS_TO_CAPTURE) {
+          toReturn = function () {
+            const function_value = FUNCTION_CALL_ARGUMENTS_TO_CAPTURE[pathname].apply(this, arguments);
+            if (!httparchive_observers.function_values[pathname][function_value]) {
+              httparchive_observers.function_values[pathname][function_value] = 0;
+            }
+            httparchive_observers.function_values[pathname][function_value]++;
+            return original.apply(this, arguments);
+          }
+        }
         // Increment the feature counter.
         httparchive_observers[pathname]++;
```

Comment:
WDYT about something like this, to avoid needing to alias `toReturn = original` above?

Swapped the if statement with the increment statement and immediately returned the wrapper function.

```suggestion
        // Increment the feature counter.
        httparchive_observers[pathname]++;
        
        if (pathname in FUNCTION_CALL_ARGUMENTS_TO_CAPTURE) {
          return function () {
            const function_value = FUNCTION_CALL_ARGUMENTS_TO_CAPTURE[pathname].apply(this, arguments);
            if (!httparchive_observers.function_values[pathname][function_value]) {
              httparchive_observers.function_values[pathname][function_value] = 0;
            }
            httparchive_observers.function_values[pathname][function_value]++;
            return original.apply(this, arguments);
          }
        }
```